bin/segment2.c

```diff
@@ -3318,162 +3371,1063 @@ const s16 seg2_painting_triangle_mesh[] = {
+      74,   84,   86,
+      74,   86,  136,
+      72,   74,  136,
+      85,  136,   86,
+      86,   48,   85,
+      48,   86,   82,
+      25,   46,   79,
+      79,   47,   25,
+      82,   46,   31,
+      19,   47,   76,
+      76,  143,   19,
+      31,   48,   82,
+      37,   48,  146,
+      85,   48,   37,
+      10,   87,   11,
+      87,   10,  147,
+      92,   95,  149,
+      88,   89,   90,
+      89,  148,   91,
+      90,   89,   91,
+      91,   92,   90,
+      92,  149,   90,
+      93,   87,   94,
+      87,   93,   11,
+      94,   87,   95,
+      87,  147,   95,
+      95,   92,   94,
+      96,   92,   91,
+      92,   96,   94,
+      39,  101,   97,
+      97,   98,   37,
+      98,   97,  101,
+      99,   98,  103,
+      99,  150,   98,
+      37,   98,  150,
+      98,  101,  103,
+     100,  103,  101,
+     101,  102,  100,
+     102,  101,   39,
+     100,  107,  103,
+     103,  104,   99,
+     104,  103,  107,
+     105,  104,  109,
+     105,  151,  104,
+      99,  104,  151,
+     104,  107,  109,
+     106,  109,  107,
+     107,  108,  106,
+     108,  107,  100,
+     109,  110,  105,
+     106,  152,  109,
+     110,  109,  152,
+     105,  110,  153,
+     111,  152,  106,
+      11,   93,   45,
+     102,   45,   93,
+      45,  102,   39,
+     102,   93,  112,
+     100,  102,  112,
+      94,  112,   93,
+     112,  108,  100,
+     108,  112,   96,
+      94,   96,  112,
+     106,  108,  113,
+     108,   96,  113,
+      91,  113,   96,
+      91,  148,  113,
+     113,  111,  106,
+     111,  113,  148,
+     114,  116,   99,
+      99,  115,  114,
+     115,   99,  151,
+      99,  116,  150,
+      72,  124,  120,
+     116,   37,  150,
+      37,  116,   85,
+     117,  105,  153,
+     105,  115,  151,
+     105,  117,  118,
+     118,  115,  105,
+     119,  120,  124,
+     120,  119,   70,
+     119,  124,  127,
+     119,  121,   70,
+     121,  119,  122,
+     122,  119,  127,
+     123,  124,   72,
+     124,  123,  125,
+     125,  127,  124,
+     126,  127,  131,
+     127,  126,  122,
+     125,  131,  127,
+     126,  131,  134,
+     128,  126,  129,
+     129,  126,  134,
+     126,  128,  122,
+     136,  123,   72,
+     130,  134,  131,
+     131,  132,  130,
+     132,  131,  125,
+     133,  134,  155,
+     134,  133,  129,
+     130,  155,  134,
+     133,  156,  129,
+     135,  155,  130,
+     123,  136,  137,
+      85,  137,  136,
+     139,  115,  118,
+     123,  137,  138,
+     125,  123,  138,
+     114,  138,  137,
+     137,  116,  114,
+     116,  137,   85,
+     114,  139,  138,
+     132,  138,  139,
+     138,  132,  125,
+     132,  139,  140,
+     130,  132,  140,
+     115,  139,  114,
+     118,  140,  139,
+     135,  140,  154,
+     118,  154,  140,
+     140,  135,  130,
+     117,  154,  118,
+};
+
+// 0x07023044 - 0x07023048
+const PaintingData *const seg2_painting_env_map_texture_maps[] = {
+    seg2_painting_env_map_texture_map,
```

Comment:
there is a LOT of extra data being added to seg2 here. i wonder if it may be a good idea to start splitting this into different files based on purpose, to make finding the relevant stuff easier?

Reply:
Most of that extra data is now removed, now only one list of vertices and one list of triangles is used for everything. We should still split up segment2.c in the future though.